src/ShortcutGuide.tsx

```diff
@@ -22,7 +22,7 @@ const ShortcutGuide: React.FC = () => {
 
 	return (
 		<>
-			<div className="a11y_modal" data-testid="a11y_modal" role="button">
+			<div className="a11y_modal" data-testId="a11y_modal" role="button">
```

Comment:
Don't change the testids to this format, because that's not common convention in regards to data attributes.

The accepted common names are:
- `data-test`
- `data-testid`
- `data-cy` (cypress only)

Also, and more important, why are we hijacking the role of the div element with a role of button?
What is preventing us from using a native button element?

Because even with the role of button, this element is not capable of receiving focus, as there's no tabindex value here.

My suggestions are:
1. Change the data attribute back to the convention
2. Avoid adding custom semantics. It might require a refactor of the html, but it's less work, trust me.